test/blackbox-tests/test-cases/variants/run.t

```diff
@@ -266,6 +266,26 @@ Then we make sure that we can implement it
   File "impl-lib/dune", line 3, characters 13-17:
   3 |  (implements vlib))
                    ^^^^
-  Error: Library "vlib" is not virtual. It cannot be implemented by "impl".
-  -> required by library "impl" in _build/default/impl-lib
+  Error: It's not possible to implement external libraries yet
   [1]
+
+Include variants and implementation information in dune-package
+  $ dune build --root dune-package-info
+  Entering directory 'dune-package-info'
+  (lang dune 1.6)
+  (name foo)
+  (library
+   (name foo.impl)
+   (kind normal)
+   (archives (byte impl/impl.cma) (native impl/impl.cmxa))
+   (plugins (byte impl/impl.cma) (native impl/impl.cmxs))
+   (foreign_archives (native impl/impl$ext_lib))
+   (requires foo.vlib)
+   (implements vlib)
+   (main_module_name Vlib))
+  (library
+   (name foo.vlib)
+   (kind normal)
+   (foreign_archives (native vlib/vlib$ext_lib))
+   (main_module_name Vlib)
+   (virtual (modules ((name Vmod) (visibility public) (virtual_ true)))))
```

Comment:
Is this listing only the virtual modules? If yes, we might as well skip the `virtual_ true`. In any case, we don't need the trailing underscore.

Reply:
`virtual` speifies that it's listing extra information for virtual libraries. For now it's only the `modules` field (which contains all modules). I could remove the `modules` level of indirection. But I thought it would be nice as it would allow us to add new fields easily.